### PR TITLE
This pull request is to connect external stylesheet and script file

### PR DESCRIPTION
**Pull request by AnitaDames**
## Connect stylesheet and script file:

Add
- Boilerplate to makeup

- Link stylesheet to index.html and tested that stylesheet is connected
- Link external JavaScript to index.html and tested that it is connected

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Neiva Skyline</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <h1>Welcome</h1>
+    <script src="app.js"></script>
```
Comment (steveWDamesJr) [inline index.html:12]: Great job implementing the boilerplate for the markup! 💯

File: index.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+h1{
+    color: brown;
+}
```
Comment (steveWDamesJr) [inline index.css:3]: Excellent job Connecting and testing external stylesheet file 🥇

File: app.js [hunk 3]
```diff
@@ -0,0 +1 @@
+console.log("Hello World")
```
Comment (steveWDamesJr) [inline app.js:1]: Great job connecting and testing script file 🎉

**Review by steveWDamesJr — APPROVED** [review]
Hi @TeacherAnita  

Excellent job!

✔️ Great job connecting external stylesheet 💯 
✔️ Great job connecting external script file 💯 
✔️ Awesome implementation of tests 💯 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

Approved emoji

Cheers and Happy coding!👏👏👏

**Comment by AnitaDames** [comment]
Thank You!

Happy Coding 🎉

**Pull request merged**